### PR TITLE
[Contributing.md] Suggestion to improve the contribution best practices

### PR DESCRIPTION
**Pull request by cwilso**
This is merely a suggestion to improve the contribution best practices to reflect (what I think is) the actual status quo.

This is based on an attempt to figure out how we could have avoided some incidents that ruffled a few feathers, by being clearer about expectations;  I'd welcome feedback about whether this is useful or if it could be improved in some other way.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,11 +39,29 @@ issues on a regular basis; however note that sometimes this can take awhile
...
+the WG's attention is needed to move forward, ask one of the Editors to flag it.
+
+In general, you should not directly commit changes to a draft unless you are an
+Editor of that draft, or have their explicit permission.  If you are not an 
+Editor of a draft, but wish to contribute changes, the best practice is to either 
+work directly with an Editor to review proposed text, or file your proposal as a 
+pull request (PR). Substantive changes need WG consensus, not merely Editor 
+agreement; typographic error and markup fixes are generally okay to commit, but 
+any substantive changes, or even substantial changes/additions to non-normative
+text, should have clear WG consensus.
```
Comment (fantasai) [inline CONTRIBUTING.md:55]: Non-normative text doesn't generally require WG consensus, only if it's reflecting something sensitive. Substantial changes to non-normative text should have the editors approval though.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -39,11 +39,29 @@ issues on a regular basis; however note that sometimes this can take awhile
...
+
+In any case, WG consensus is expected prior to merging changes, and consensus is 
+determined by the Chairs (not self-assessed) via synchronous decisions during 
+meetings, and occasionally via async CFCs. Some degree of discretion is afforded 
+to Editors to make changes prior to WG consensus, particularly early in a spec or 
+feature's lifecycle, although Editors should confirm those changes with the WG
```
Comment (fantasai) [inline CONTRIBUTING.md:61]: s/should/must/

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -39,11 +39,29 @@ issues on a regular basis; however note that sometimes this can take awhile
...
+text, should have clear WG consensus.
+
+In any case, WG consensus is expected prior to merging changes, and consensus is 
+determined by the Chairs (not self-assessed) via synchronous decisions during 
+meetings, and occasionally via async CFCs. Some degree of discretion is afforded 
+to Editors to make changes prior to WG consensus, particularly early in a spec or 
+feature's lifecycle, although Editors should confirm those changes with the WG 
+prior to republishing on TR.  References to the discussion and the WG consensus 
+should be placed in the issue or commit.  Agreement from an Editor in an issue 
+is not a replacement for WG consensus.
```
Comment (fantasai) [inline CONTRIBUTING.md:64]: s/replacement/substitute/

File: CONTRIBUTING.md (hunk 1 above)
Comment (cwilso) [reply CONTRIBUTING.md]: Addressed.

File: CONTRIBUTING.md (hunk 3 above)
Comment (cwilso) [reply CONTRIBUTING.md]: Changed.

**Review by frivoal — APPROVED** [review]

**Review by hober — APPROVED** [review]
Looks good to me!

**Comment by cwilso** [comment]
As this isn't a spec, unfortunately these guidelines aren't quite applicable (since there's no Editor :P ).  As I note a couple of approvals and a thumbs-up from one of the chairs, I'll leave it to you all to decide when it's appropriate to merge once @fantasai has given it a re-review.

**Review by tantek — APPROVED** [review]
LGTM, this is consistent with my rough understanding of current WG expectations.

**Comment by astearns** [comment]
We will announce this update on the Feb 28 call and can take further improvements

**Comment by bramus** [comment]
To those following: filed a small PR to clarify where PRs should originate from: https://github.com/w3c/csswg-drafts/pull/10009

**Pull request merged**